<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PwdSet</class>
 <widget class="QDialog" name="PwdSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create pwd</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="ContentVL">
     <item>
      <widget class="QLabel" name="InfoLbl">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create password for new user</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="FormVL">
       <item>
        <layout class="QVBoxLayout" name="EditVL">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="PwdLbl">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="ConfirmLbl">
             <property name="text">
              <string>Confirm</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="ConfirmLEdit">
             <property name="inputMethodHints">
              <set>Qt::ImhHiddenText</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="PwdLEdit">
             <property name="inputMethodHints">
              <set>Qt::ImhHiddenText|Qt::ImhSensitiveData</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="NameLEdit"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="NameLbl">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="YN">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>YN</sender>
   <signal>accepted()</signal>
   <receiver>PwdSet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YN</sender>
   <signal>rejected()</signal>
   <receiver>PwdSet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
